src/coreclr/src/tools/r2rdump/R2RDiff.cs

```diff
@@ -349,6 +381,20 @@ private void DumpCommonMethods(Dumper dumper, int moduleIndex, Dictionary<string
             }
         }
 
+        private static Dictionary<string, ReadyToRunImportSection.ImportSectionEntry> GetImports(ReadyToRunReader reader)
+        {
+            Dictionary<string, ReadyToRunImportSection.ImportSectionEntry> result = new Dictionary<string, ReadyToRunImportSection.ImportSectionEntry>();
+            SignatureFormattingOptions signatureOptions = new SignatureFormattingOptions() { Naked = true };
```

Comment:
I think it is one of the cases when using `var` on the left side would improve readability.

Reply:
Sounds reasonable, fixed in 2nd commit.